code/workspaces/web-app/src/components/app/Navigation.js

```diff
@@ -4,46 +4,41 @@ import { withStyles } from '@material-ui/core/styles';
 import SideBar, { drawerWidth } from './SideBar';
 import TopBar from './TopBar';
 
-const topbarHeightLarge = 64;
-const topbarHeightSmall = 56;
-const topbarHeightXSmall = 48;
-
 const styles = theme => ({
   container: {
     backgroundColor: theme.palette.backgroundColor,
     width: '100%',
-    position: 'absolute',
-    top: 0,
-    bottom: 0,
+    height: `calc(100vh - ${theme.shape.topBarHeight}px)`,
+    paddingTop: theme.shape.topBarHeight,
   },
   appFrame: {
     backgroundColor: theme.palette.backgroundColor,
-    display: 'flex',
     width: '100%',
-    position: 'relative',
+    height: '100%',
     zIndex: 1,
   },
-  content: {
+  page: {
     width: `calc(100% - ${drawerWidth}px)`,
-    marginLeft: drawerWidth,
-    '@media (min-width:1px) and (orientation: landscape)': {
-      marginTop: topbarHeightXSmall,
-    },
-    '@media (min-width:600px)': {
-      marginTop: topbarHeightLarge,
-    },
-    marginTop: topbarHeightSmall,
+    maxWidth: '750px',
```

Comment:
This feels too narrow on a desktop monitor when using cards. We might need to leave this wider, maybe 1000px?

Reply:
Good point. I went for this width as I didn't know whether we were moving to the more row like cards as in the designs I proposed or whether we are leaving the cards as is for notebooks etc?